modules/customize-certificates-view-service-serving.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="view-service-serving-{context}"]
+= View the expiration date of the generated service certificate
+
+While the certificate and key pair is automatically replaced when
+it gets close to expiration, it is possible to view the expiration
```

Comment:
~~it is possible to view the expiration date within the secret.~~ you can view the secret's expiration date.

Why would I want to do this?

Reply:
I think you're right on this module. Originally I was thinking it was so you could view the expiration date of the certificate, but the only expiration date that matters is the service CA, which is handled in a separate module.

I'm deleting this file and removing the reference.